apollo-router/src/configuration/mod.rs

```diff
@@ -316,16 +358,24 @@ impl Configuration {
                 plugins: apollo_plugins,
             },
             dev,
-        }
+        };
+
+        configuration.validate()
+    }
+}
+
+impl Configuration {
+    fn validate(self) -> Result<Self, ConfigurationError> {
+        Ok(self)
```

Comment:
Always valid then ?

Reply:
This will change [soon](https://github.com/apollographql/router/pull/1796/files#diff-4a11c81ac00715f2740e504e8041ac1fe77d97b65f6d367a4fff4fe269e2114eR370)! :D